e2e/features/step_definitions/list.step.js

```diff
@@ -26,3 +26,10 @@ Then(/^I expect list created in board$/, (dataTable) => {
   let titleString = rHash.Title;
   expect(dashboard.isCreatedList(titleString)).to.be.true;
 });
+
+Given(/^I select a list with:$/, (data) => {
+  dashboard = new Dashboard();
```

Comment:
I know this is not your code, but we should rename this all variables dashboard by board.

Reply:
I will refactor it in a new branch.